### ISSUE TITLE
Add media control section

### ISSUE THREAD

**Issue by Boring-student** [iamrasel/lunar-launcher#40]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/iamrasel/lunar-launcher/issues) or [closed](https://github.com/iamrasel/lunar-launcher/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have already checked the [WIKI](https://github.com/iamrasel/lunar-launcher/wiki) pages and made sure that the feature isn't already exist.
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.

### Feature description

Basically when media is playing the media control section should slide down or pop up that let you control media that's currently playing somewhat like this.
![image](https://user-images.githubusercontent.com/74827801/209898830-d662ac41-50c4-455f-bb02-6d2b613b308c.png)

**Comment by iamrasel** [comment]
It would be awesome, so I might work on this later someday.

**Comment by michelevantaggi02** [comment]
i found [this](https://stackoverflow.com/questions/51326579/control-another-apps-audio-playback), maybe it can be useful
